src/components/ActionBar/README.md

```diff
@@ -8,8 +177,44 @@
 | default | —           |
 
 
+## ActionBarButton Props
+
+Supports attributes from [`<button>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button).
+
+| Prop       | Type      | Default     | Possible values                    | Description                           |
+| ---------- | --------- | ----------- | ---------------------------------- | ------------------------------------- |
+| type       | `string`  | `'button'`  | —                                  | Type of the button                    |
+| full-width | `boolean` | `false`     | —                                  | Whether to make the button full-width |
+| color      | `string`  | `'#000'`    | —                                  | Background color of button            |
+| text-color | `string`  | —           | —                                  | Text color of button                  |
+| variant    | `string`  | `'primary'` | `primary`, `secondary`, `tertiary` | Semantic variant                      |
```

Comment:
I think we need to reduce this to just primary / secondary.  There's only two states in actionbar buttons.

Reply:
removed